indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -8,7 +8,7 @@
    */
   Indigo.AnnotationView = Backbone.View.extend({
     className: function() {
-      return 'annotation ' + (!this.model.get('in_reply_to') ? 'root' : 'reply');
+      return 'annotation ' + (!this.model.get('in_reply_to') ? 'root' : 'reply_id');
```

Comment:
This seems odd to me? Does this really need to change?

Reply:
I forgot to revert this change, at some point fake annotations manifested on the front-end as empty annotation forms with the empty text field and the buttons, I was trying to isolate the issue.

I am reverting it.